build.proj

```diff
@@ -36,6 +36,7 @@
       <_ProjectsToPackage Include="$(MSBuildThisFileDirectory)src\Iot.Device.Bindings\Iot.Device.Bindings.csproj" />
       <_ProjectsToPackage Include="$(MSBuildThisFileDirectory)src\System.Device.Gpio\System.Device.Gpio.csproj" />
       <_ProjectsToPackage Include="$(MSBuildThisFileDirectory)src\Iot.Device.Bindings.SkiaSharpAdapter\Iot.Device.Bindings.SkiaSharpAdapter.csproj" />
+      <_ProjectsToPackage Include="$(MSBuildThisFileDirectory)tools\ArduinoCsCompiler\Frontend\Frontend.csproj"/>
```

Comment:
This will make it so that this tool is marked as shipping, signed, etc. Do we actually need to produce the package from here? Given this tool is more of a "tool that we have inside our repo for your benefit" kind of thing, I'd rather have the scripts call pack into the project directly.

Reply:
I have nothing against marking the tool as isPackable, but we don't really have a good mechanism today for shipping tools that are just utilities and not really supported from our side, so I would suggest that for having folks using the tool, we keep the installation vehicle to be "run a script which will build the project, pack it, and install it for you so you can use it" kind of thing.

Reply:
Already discussed: It is required that the tool links to the officially released versions of Iot.Device.Bindings and System.Device.Bindings, otherwise the reference handling during compilation will not work.